### ISSUE TITLE
fmt 9.0 released with breaking API changes

### ISSUE THREAD

**Issue by musicinmybrain** [jpcima/fmidi#5]
See: https://github.com/fmtlib/fmt/releases/tag/9.0.0

Currently, with fmt 9.0, building `fmidi` fails with errors like (abbreviated):

```
/builddir/build/BUILD/fmidi-0.1.1/programs/midi-seq.cc:33:19:   required from here
/usr/include/fmt/core.h:1733:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1733 |       formattable,
      |       ^~~~~~~~~~~
/usr/include/fmt/core.h:1733:7: note: 'formattable' evaluates to false
```

A workaround is to define the `FMT_DEPRECATED_OSTREAM` preprocessor macro, but this will be removed in fmt 10.0.

**Comment by jpcima** [comment]
Thanks for the notice.
It is fixed in revision 2091642.

**Comment by musicinmybrain** [comment]
> Thanks for the notice. It is fixed in revision [2091642](https://github.com/jpcima/fmidi/commit/20916421656e89a1303a85e76e89db8bd551a31e).

Thank you! That works wonderfully.

**Comment by jpcima** [comment]
A problem is that this fix breaks with fmt < 9.0.0
Perhaps I should implement a custom formatter.
Or try getting rid of fmt altogether...
